Example/Example/CharactersPresenter.swift

```diff
@@ -49,7 +49,7 @@ class CharactersPresenter: BothamPresenter, BothamPullToRefreshPresenter {
             Character(name: "BlackPanther", thumbnail: NSURL(string: "https://i.annihil.us/u/prod/marvel/i/mg/9/03/537ba26276348.jpg")!),
             ])
 
-        self.ui?.wilEndRefreshing()
+        self.ui?.stopRefreshing()
```

Comment:
I've renamed this method to match with the name convention we had in the loading view ``showLoading`` ``hideLoading`` where we use orders. The previous name had also a typo :)